nilearn/tests/test_masking.py

```diff
@@ -384,6 +411,15 @@ def test_compute_multi_epi_mask():
     assert_array_equal(mask_ab, mask_ab_.get_data())
 
 
+def test_compute_multi_gray_matter_mask():
+    assert_raises(TypeError, compute_multi_gray_matter_mask, [])
+
+    # Check error raised if images with different shapes are given as input
+    imgs = [Nifti1Image(np.ones((9, 9, 9)), np.eye(4)),
+            Nifti1Image(np.ones((9, 9, 8)), np.eye(4))]
+    assert_raises(ValueError, compute_multi_gray_matter_mask, imgs)
```

Comment:
Please add tests to compute the mask on multiple images ? There are tests covering only the error messages.